### PR TITLE
[16.0] [MIG] hr_timesheet_sheet_current

### PR DESCRIPTION
**Pull request by victor-champonnois**
https://gestion.coopiteasy.be/web#id=10959&action=475&active_id=492&model=project.task&view_type=form&menu_id=

### CODE REVIEW

File: hr_timesheet_sheet_current/__manifest__.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
     "version": "12.0.1.0.0",
     "license": "AGPL-3",
     "author": "Coop IT Easy SC",
-    "website": "https://coopiteasy.be",
+    "website": "https://github.com/coopiteasy/cie-timesheet",
```
Comment (huguesdk) [inline hr_timesheet_sheet_current/__manifest__.py]: i don’t think that this should be changed. the github repo’s url is used only for oca repos. this should be fixed in `.copier-answers.yml`.

**Review by huguesdk — DISMISSED** [review]
just a misc change request, otherwise lgtm.

File: hr_timesheet_sheet_current/__manifest__.py (hunk 1 above)
Comment (victor-champonnois) [reply hr_timesheet_sheet_current/__manifest__.py]: fixed (in module and in copier)

**Review by carmenbianca — APPROVED** [review]
LGTM

**Pull request merged**